### DISCUSSION TITLE
Forged recipients and sender recognised on alias+srs

### DISCUSSION THREAD

**Discussion by thegcat** [rspamd/rspamd#3786] [Q&A] [answered]
Hello,

We currently have emails triggering the `FORGED_RECIPIENTS` and `FORGED_SENDER` for emails that are received and forwarded by another mail host to mailing lists we host. The two headers (slightly redacted) are:

```
FORGED_RECIPIENTS (2) [m:buero@die-koma.org,s:SomeList@lists.fachschaften.org]
FORGED_SENDER (0.3) [info@abruesten.jetzt,SRS0=p1jc=LA=abruesten.jetzt=no-answer@TheForwardingMailHost]
```

Is there any way to whitelist parts of that stack so that those rules don't apply in those cases? Unfortunately some emails have other high-hitting symbols that bring those valid emails over our current reject score.

Thanks for any pointers in this case.

**Comment by fatalbanana** [comment, marked as answer]
If it doesn't suit to reduce scoring globally you could add exclusions using [composites](https://rspamd.com/doc/configuration/composites.html#composites-on-symbol-options) (or alternatively [multimap](https://rspamd.com/doc/modules/multimap.html#symbol-options-filters)). If it suits to disable/rescore one/more symbols (or disable reject action?) for a particular recipient you might consider using [user settings](https://rspamd.com/doc/configuration/settings.html).

~~~
# /etc/rspamd/local.d/composites.conf
BALANCED_FORGED_RECIPIENTS {
  expression = "FORGED_RECIPIENTS[s:SomeList@lists.fachschaften.org]";
  policy = "remove_weight";
  score = 0.0;
}
~~~

**Reply by vstakhov** [reply]
I think this issue has been fixed long time ago.

**Reply by thegcat** [reply]
> If it doesn't suit to reduce scoring globally you could add exclusions using [composites](https://rspamd.com/doc/configuration/composites.html#composites-on-symbol-options) (or alternatively [multimap](https://rspamd.com/doc/modules/multimap.html#symbol-options-filters)). If it suits to disable/rescore one/more symbols (or disable reject action?) for a particular recipient you might consider using [user settings](https://rspamd.com/doc/configuration/settings.html).

I wasn't aware of both those configuration options, thank you! We should be able to work around this issue that way.

**Reply by thegcat** [reply]
> I think this issue has been fixed long time ago.

If it helps we use rspamd 2.7 on Ubuntu from the rspamd apt repos.
